### PR TITLE
Create FCP Site

### LINKED ISSUE #93: Update GAM config for channelized ad units
Also create ad units in GAM

### LINKED ISSUE #94: Re-create Spec Guides

### LINKED ISSUE #95: Create Directory landing page

### LINKED ISSUE #96: Create Economics channel layout
Lift from OEM

### LINKED ISSUE #97: Asphalt Industry Directory
Iframe from somewhere

### PR DESCRIPTION
**Pull request by solocommand**
Closes #94 - Spec Guides
screencapture-0-0-0-0-9716-specguide-2020-03-27-13_47_59
screencapture-0-0-0-0-9716-specguide-asphalt-paver-2020-03-27-13_49_39

Closes #97 - Asphalt Industry Directory
asphalt-industry-directory

Closes #96 - Economics custom layout
screencapture-0-0-0-0-9716-economics-2020-03-26-13_35_48

Closes #93 - GAM config

Closes #95 - Directory
screencapture-0-0-0-0-9716-directory-2020-03-27-13_43_32

### CODE REVIEW

**Comment by solocommand** [comment]
I guess Travis is being dumb or something. Build for 4b370c7 passed: https://travis-ci.org/github/base-cms-websites/ac-business-media/builds/667841864

File: packages/directory/components/components/list-node/list-node.marko [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { getAsObject } from "@base-cms/object-path";
```
Comment (zarathustra323) [inline packages/directory/components/components/list-node/list-node.marko]: All of the list node components should move to the `refresh-theme` package, not live in directory. Directory should only have "self-contained" components. Look at how the `directory` and `shared` packages are working with each other in endeavor

File: packages/refresh-theme/components/nodes/spec-guide-card.marko [hunk 2]
```diff
@@ -49,7 +49,7 @@ $ const buttonClasses = `spec-guide-button spec-guide-button--${node.alias}`;
         class=buttonClasses
         href=content.siteContext.path
       >
-        Visit the Interactive $!{content.shortName}
+        Visit the Interactive<br>$!{content.shortName}
```
Comment (zarathustra323) [inline packages/refresh-theme/components/nodes/spec-guide-card.marko]: Are these `<br>` tags necessary? Why can't the content simply wrap inside the button?

File: sites/forconstructionpros.com/config/site.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      bgColor: '#000',
+      logo: 'https://base.imgix.net/files/base/acbm/fcp/image/static/logo/site_logo.png?h=45',
+    },
+    to: 'nraether@acbusinessmedia.com',
+  },
+  inquiry: {
+    enabled: true,
+    directSend: true,
+    sendTo: 'nraether@acbusinessmedia.com',
+    sendFrom: 'OEM Off-Highway <noreply@baseplatform.io>',
```
Comment (zarathustra323) [inline sites/forconstructionpros.com/config/site.js]: Wrong send from info.

Can you verify all the other IDs in this config are for FCP and not OEM?

File: packages/refresh-theme/templates/spec-guide/guide.marko [hunk 4]
```diff
@@ -57,6 +57,9 @@ $ const adSlots = () => ({
                   </if>
                 </ul>
               </if>
+              <if(config.seeAlso)>
+                <p>$!{config.seeAlso}</p>
```
Comment (zarathustra323) [inline packages/refresh-theme/templates/spec-guide/guide.marko]: The `seeAlso` in the config is already using a `<p>` element. Paragraphs shouldn't have paragraphs inside them. As such, change this to `<div>`

**Pull request merged**